modules/logging-release-notes-5-7-12.adoc

```diff
@@ -0,0 +1,53 @@
+// module included in logging/logging-5-7-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-7-12_{context}"]
+= Logging 5.7.12
+This release includes link:https://access.redhat.com/errata/RHSA-2024:1508[Logging Bug Fix 5.7.12].
```

Comment:
Just curious (as I do not know about this particular feature). Why doesn't this RN (and the previous RN) include "OpenShift" in title description but the earlier RNs do? 
![image](https://github.com/openshift/openshift-docs/assets/122639474/82a78de6-0960-4864-83a8-aeef2b01e931)

Reply:
This is probably because another writer worked on 5.7.10, also all the previous releases were drafted by me. :D I'll make the correction.